### PR TITLE
Initial server code

### PR DESCRIPTION
**Pull request by janathsrikrishnan**
1. initialized golang package
2. created skeleton server

### CODE REVIEW

**Comment by kamalraj-ten** [comment]
Add a readme file describing how to run the service and contact it

File: service/readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+To initialize the service run this cmd
```
Comment (kamalraj-ten) [inline service/readme.md:1]: change the name to README.md

File: service/readme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+```
+cd service
+go run main.go
+```
```
Comment (kamalraj-ten) [inline service/readme.md:16]: you forgot to mention how to contact it, eg `curl`

File: service/readme.md (hunk 1 above)
Comment (janathsrikrishnan) [reply service/readme.md:1]: done

File: service/readme.md (hunk 2 above)
Comment (janathsrikrishnan) [reply service/readme.md:16]: done

**Review by kamalraj-ten — APPROVED** [review]

**Pull request merged**